x-pack/plugins/cases/server/services/attachments/index.ts

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Logger, SavedObject, SavedObjectReference } from 'kibana/server';
+
+import {
+  CommentAttributes as AttachmentAttributes,
+  CommentPatchAttributes as AttachmentPatchAttributes,
+} from '../../../common/api';
+import { CASE_COMMENT_SAVED_OBJECT } from '../../../common/constants';
+import { ClientArgs } from '..';
+
+interface GetAttachmentArgs extends ClientArgs {
+  attachmentId: string;
+}
+
+interface CreateAttachmentArgs extends ClientArgs {
+  attributes: AttachmentAttributes;
+  references: SavedObjectReference[];
+  id: string;
+}
+
+interface UpdateArgs {
+  attachmentId: string;
+  updatedAttributes: AttachmentPatchAttributes;
+  version?: string;
+}
+
+type UpdateAttachmentArgs = UpdateArgs & ClientArgs;
+
+interface BulkUpdateAttachmentArgs extends ClientArgs {
+  comments: UpdateArgs[];
+}
+
+export class AttachmentService {
+  constructor(private readonly log: Logger) {}
+
+  public async get({
+    unsecuredSavedObjectsClient,
+    attachmentId,
+  }: GetAttachmentArgs): Promise<SavedObject<AttachmentAttributes>> {
+    try {
+      this.log.debug(`Attempting to GET attachment ${attachmentId}`);
+      return await unsecuredSavedObjectsClient.get<AttachmentAttributes>(
+        CASE_COMMENT_SAVED_OBJECT,
+        attachmentId
+      );
+    } catch (error) {
+      this.log.error(`Error on GET attachment ${attachmentId}: ${error}`);
+      throw error;
+    }
+  }
+
+  public async delete({ unsecuredSavedObjectsClient, attachmentId }: GetAttachmentArgs) {
+    try {
+      this.log.debug(`Attempting to GET attachment ${attachmentId}`);
```

Comment:
GET => DELETE in these logs

Reply:
Fixed in https://github.com/elastic/kibana/pull/101465